bundles/net.tourbook.device.garmin.fit/src/net/tourbook/device/garmin/fit/listeners/MesgListener_DeviceInfo.java

```diff
@@ -31,8 +31,9 @@
 
 public class MesgListener_DeviceInfo extends AbstractMesgListener implements DeviceInfoMesgListener {
 
+   private static final String            _10S                = " %10s";                                          //$NON-NLS-1$
    private static final String            FORMAT_FLOAT_10_3   = "%10.3f";                                         //$NON-NLS-1$
```

Comment:
Can you rename `_10S` into `FORMAT_STRING_10` like `FORMAT_FLOAT_10_3`

Reply:
Fixed